pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.shanerx</groupId>
     <artifactId>tradeshop</artifactId>
-    <version>1.6.0-BETA</version>
+    <version>1.6.0-Updater</version>
```

Comment:
What's this? This isn't a valid release tag. I assume you wanted to check wether the latest build had been compiled...?
If yes, this should be changed back to BETA.

Reply:
Correct, and I edited and pushed right before I left my house should already be fixed